packages/edit-site/src/components/navigation-sidebar/navigation-panel/style.scss

```diff
@@ -96,18 +94,33 @@
 		text-align: left;
 		padding-left: $grid-unit-20;
 		padding-right: $grid-unit-20;
-		margin-bottom: $grid-unit-15;
 		color: inherit;
 	}
 }
 
 .edit-site-navigation-panel__template-item-title {
+	font-size: 14px;
+	line-height: 20px;
+
 	em {
 		margin-right: 1ch;
 	}
 }
+
+.components-navigation__item + .edit-site-navigation-panel__template-item {
+	margin-top: $grid-unit-20;
+}
+
+.edit-site-navigation-panel__template-item + .edit-site-navigation-panel__template-item {
+	margin-top: $grid-unit-10;
+}
+
+.components-menu-item__info-wrapper {
+	padding: $grid-unit-05 0;
+}
```

Comment:
Again, a little odd we are adding styles for this classname here since it corresponds to MenuItem in the interface package.  If we need to add a custom wrapper we should probably call it `edit-site-navigation-panel__info-wrapper`

Reply:
Good spot. Updated.

Reply:
oops! My apologies for not responding sooner.  It looks good to me.